Source/abci/Importer/aiObject.cpp

```diff
@@ -58,8 +58,13 @@ aiObject::aiObject(aiContext *ctx, aiObject *parent, const abcObject &abc)
     , m_abc(abc)
     , m_parent(parent)
 {
+#if defined(__linux__)
```

Comment:
Would have been nice to have this in a separate commit

Reply:
Done